framework/src/wifi_manager/wifi_manager.c

```diff
@@ -186,7 +186,7 @@ static void wifi_linkdown_event_func(void)
 	if (g_manager_info.mode == STA_MODE) {
 		nvdbg("WIFI DISCONNECTED AP - STA MODE");
 		strncpy(g_manager_info.ssid, "", 32);
-		strcpy(g_manager_info.ip4_address, "");
+		strncpy(g_manager_info.ip4_address, "", 18);
```

Comment:
I think below is enough to initialize, strncpy is too big functionality.
```
g_manager_info.ip4_address[0] = '\0';
```
How about change it?

Reply:
Done